omas/omas_physics.py

```diff
@@ -227,15 +291,27 @@ def map_flux_coordinate_to_pol_flux(ods, time_index, origin, values):
 
     :return: Transformed values
     """
-    if origin == "rho_pol":
-        return (
-            values**2
-            * (
-                ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_sep"]
-                - ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_axis"]
-            )
-            + ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_axis"]
-        )
+    if origin == "rho_pol_norm":
+        return (values**2 * (ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_boundary"]
+                - ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_axis"])
+                + ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_axis"])
+    elif origin == "rho_tor_norm":
+        phi = values**2
+        phi *= np.array([ods["equilibrium"]["time_slice"][time_index]["global_quantities"]["psi_boundary"]]) / np.max(phi)
```

Comment:
Phi is normalized or real poloidal flux? Because you should multiply it by `phi_boundary` not by `psi_boundary`
also, why are you normalizing it by np.max(phi)? What it somebody use as values rho_tor = 0.5? It will force it to be one, always.